shell.nix

```diff
@@ -1,3 +1,39 @@
-(import (fetchTarball https://github.com/edolstra/flake-compat/archive/master.tar.gz) {
```

Comment:
can we keep the original shell.nix and default.nix, those are there for compatibility reasons with non flake enabled systems.

Reply:
default files restored